SConstruct

```diff
@@ -297,6 +297,13 @@ options.Add(
 	"usd_"
 )
 
+options.Add(
+	"USD_LIB_SUFFIXES",
+	"The suffixes to append to the names of the USD libraries. You can modify this "
+	"to \"usd_ms\" to use a monolithic build of USD. ",
+	"sdf arch tf vt"
+)
```

Comment:
I think this is a bit of a misleading way of solving this problem - we're going to change the list of libraries we need to link against in future, so it's not like anyone could specify anything other than `ms` here without causing future breakage. So I think a boolean `USD_MONOLITHIC` flag would be more appropriate.